teams/team-geordi/dev/templates/ho11y/ho11y.yaml

```diff
@@ -0,0 +1,154 @@
+apiVersion: v1
```

Comment:
it is code smell to put ho11y chart under templates. 

Suggested structure:

```
team-geordi/
    dev/
        Chart.yaml
        values.yaml
        templates 
             yelb-app (crd)
             ho11y-app (crd)
         ho11y (helm chart)
```

In realith ho11ly helm chart is not going to be in the repo. It will always be in the repo that belongs to team-geordi.

Reply:
Fixed this.

Reply:
In this case `ho11y-app` will point to ho11y in `\dev`